homeassistant/components/system_bridge/manifest.json

```diff
@@ -3,10 +3,10 @@
   "name": "System Bridge",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/system_bridge",
-  "requirements": ["systembridge==1.1.5"],
+  "requirements": ["systembridge==2.0.4"],
```

Comment:
Please add a link to a changelog or GitHub commit compare view for the version bump in the PR description. Thanks!

Reply:
Added to description